content/education/manage-coursework-with-github-classroom/get-started-with-github-classroom/about-assignments-in-github-classroom.md

```diff
@@ -0,0 +1,53 @@
+
+You can either create a group and let students collaborate, or create an individual assignment for each student who participates in a course on {% data variables.product.prodname_classroom %}. 
+
+When student(s) accept(s) an assignment - either individual or group -, GitHub Classroom automatically creates a new repository or adds them to a team in an existing repository. The repositories can be empty, or you can create the repositories from a template repository with starter code, documentation, tests, and other resources. Each assignment repository belongs to your organization account on {% data variables.product.prodname_dotcom %}.
+
+Each assignment has a title and an optional deadline. You can choose the visibility of repositories that GitHub Classroom creates and choose access permissions. You can also automatically grade assignments and create a dedicated space to discuss the assignment with the student.
+
+Optionally, you can enable feedback pull requests on anv assignment. If you enable this feature for an assignment, {% data variables.product.prodname_classroom %} creates a special pull request titled **Feedback** in the assignment repository for each student or group. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/leave-feedback-with-pull-requests)."
+
+If you integrate your {% data variables.product.prodname_classroom %} with an IDE, students can make changes to their assignment repositories through the IDE, or by checking out and running code locally with the appropriate software. For more information, see "[AUTOTITLE](/education/manage-coursework-with-github-classroom/integrate-github-classroom-with-an-ide/integrate-github-classroom-with-an-ide)."
+
+You can reuse existing assignments in any other classroom you have admin access to, including classrooms in a different organization. For more information, see [AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/reuse-an-assignment)."
+
+You must create a classroom before you can create an assignment. For more information, see [AUTOTITLE](/education/manage-coursework-with-github-classroom/teach-with-github-classroom/manage-classrooms#creating-a-classroom).
+
+## Types of assignments 
+
+You can either create a group assignment or n individual assignment in {% data variables.product.prodname_classroom %}.
+
+### Individual assignments
+
+An individual assignment is coursework for each student who participates in a course on GitHub Classroom. The student is intended to complete an individual assignment without help from other students.
+
+When a student accepts an assignment, GitHub Classroom automatically creates a new repository for the student. 
```

Comment:
```suggestion
{% data reusables.classroom.classroom-creates-individual-repositories %}
```

Suggestion to instead use the relevant reusable here since we're deleting that paragraph in the intro to better separate content on the different assignment types.